doc/arch/adr-011-new-app-for-ui-management.md

```diff
@@ -0,0 +1,44 @@
+# ADR 011: Create a new application for UI management 
+
+18-07-2018
+
+## Context
+
+We are building a UI to provide Content Designers and Managing Editors with 
+the data that they need to manage content at scale and make informed decisions
+around content.
+
+Now we have to make a decision about where this new functionality should live. 
+We are exploring two options:
+
+1. Create a new application to host this functionality
+2. Build the UI within the same application (Content Performance Manager)
+
+
+### Create a new application to host this functionality
+
+Pros:
+ 
+ - It better reveals the purpose of the Data Warehouse: a source of data for other applications to use
+ - It enables having a robust API so other applications could quickly benefit from having a stable API with well-defined endpoints.
+ - The architecture is easier to explain.
+ - It seems more consistent with the way GOV.UK has designed other systems.
+
+### Build the UI in the same application (Content Performance Manager) 
+
+Pros:
+
+ - There is no extra infrastructure maintenance / configuration. 
+ - The application is deployed once, there is no need to synchronise the deployments of the backend / frontend.
+ - We can evolve the application without touching the API, which impacts in having a stable API free form frequent versioning.
```

Comment:
Typo: `from` instead of `form`

Reply:
Fixed in https://github.com/alphagov/content-performance-manager/pull/798/commits/fb7e6237526b0e9e931e77afbac938c45007d2b2